frontend/src/components/molecule/DisplayQuantile.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect, useState } from 'react';
+
+import { extractor } from '../../service/fetcher';
+import { Button } from '../atomic';
+
+import { Card, Line } from '.';
+
+type DisplayProps = {
+  baseUrl: string;
+  name: string;
+  title: string;
+  timeMilliSecond: number;
+};
+
+export const DisplayQuantile: React.FC<DisplayProps> = ({
+  baseUrl,
+  name,
+  title,
+  timeMilliSecond,
+}) => {
+  const [data, setData] = useState<ReadonlyArray<number>>([]);
+  const [dataSum, setDataSum] = useState<ReadonlyArray<number>>([]);
+  const [label, setlabel] = useState<Array<string>>([]);
+  const [labelSum, setlabelSum] = useState<Array<string>>([]);
+  const InitializeClick = () => {
+    setData([]);
+    setDataSum([]);
+    setlabel([]);
+    setlabelSum([]);
+  };
+
+  useEffect(() => {
+    const quantile: Array<string> = ['0', '0.25', '0.5', '0.75', '1'];
+    const interval = setInterval(() => {
+      extractor(baseUrl, '/metrics').then(lines => {
+        if (!lines) {
+          setData(current => {
+            return [...current, 0, 0, 0, 0, 0];
+          });
+          setDataSum(current => {
+            return [...current, 0];
+          });
+        } else {
+          const index = lines.findIndex(line => line['name'] === name);
+          const tmpPath = lines[index]['metrics'][0];
```

Comment:
The example with the `tmp` var name was an **EXAMPLE**.

Reply:
Find more suitable names for your variables.

Reply:
And by the way, you're accessing object properties so, you don't need brackets around literal keys.